iidm-api/src/main/java/eu/itesla_project/iidm/network/util/Networks.java

```diff
@@ -283,11 +307,38 @@ public static void printGeneratorsSetpointDiff(Network network, Logger logger) {
     }
 
     private static double EPSILON_X = 0.01;
-
+    private static TableFormatterConfig TABLE_FORMATTER_CONFIG = TableFormatterConfig.load();
+    private static Column[] COLUMNS = {
+        new Column("id"),
+        new Column("characteristic"),
+        new Column("value")
+    };
+    
+    private static Writer createStdOutputWriter() {
+        return new OutputStreamWriter(System.out) {
+            @Override
+            public void close() throws IOException {
+                flush();
+            };
+         };
+    }
+    
     public static boolean checkFlows(String id, double r, double x, double rho1, double rho2, double u1, double u2, double theta1, double theta2, double alpha1,
-                                     double alpha2, double g1, double g2, double b1, double b2, float p1, float q1, float p2, float q2) {
+                                     double alpha2, double g1, double g2, double b1, double b2, float p1, float q1, float p2, float q2, float threshold,
+                                     boolean verbose) throws IOException {
```

Comment:
I think you have to pass the LoadFlowValidationConfig instead of verbose to:
- get the verbose option
- get the TableFormatterFactory class
- get the EPSILON_X
- get the epsilon for comparison.